app/wp-content/themes/kstg/library/style.css

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
Something isn't right here. Looks like there was an error when the css was being compiled. Not sure how things are working?